### DISCUSSION TITLE
Create a new variable for the display

### DISCUSSION THREAD

**Discussion by panhard** [dl9rdz/rdz_ttgo_sonde#480] [TTGO firmware]
Hello! Would like to ask if I want to modify the display, if I want to display a new value, for example: GPS OK, then I have to enter it in the screens2.txt file. But I don't know where the letter designations (eg: L latitade, O lOngitute, A altitude) are defined in the program? How does this part of the display work?

**Comment by dl9rdz** [comment]
This is all in Display.cpp.
This is probably best added in drawGPS(). Actually there is already a 'V' for GPS valid in there, but this is only a small symbol for the LCD displays. Something like this
```
@@ -1533,10 +1533,14 @@ void Display::drawGPS(DispEntry *de) {
        rdis->setFont(de->fmt);
        switch(de->extra[0]) {
                case 'V':
+                       // show if GPS location is valid
+                       if(!de->extra[1])
                        {
-                               // show if GPS location is valid
                                uint8_t *tile = gpsPos.valid?gps_tile:nogps_tile;
                                rdis->drawTile(de->x, de->y, 1, tile);
+                       } else {
+                               // or use de->extra+1 for the text.... just a fixed test for now
+                               drawString(de, gpsPos.valid?"GPS OK":"NO GPS");
                        }
                        break;
```

**Reply by panhard** [reply]
Thank you, I looked at Display.cpp, so I understand the program now. I want to create a new image that only has the most important displays, the position circle, the direction the probe is in. How far away the probe is, and the RX height, and the probe height. I put these in big letters on the display so that it can be seen clearly. Because my eyes are not what they used to be. :)
By the way, this program is very good and useful, I have already found many probes with its help.
I am using TTGO T-Beam V1.0 and ILI9225 with a 2inch display.
Now I am planning to build another one with an ESP32 and a 2.8inch display.
